.travis.yml

```diff
@@ -6,13 +6,10 @@ cache:
 notifications:
   email: false
 node_js:
-  - stable
-  - '0.10'
-before_install:
-  - npm i -g npm@^3.0.0
+  - 'node'
```

Comment:
eslint still runs on node 4, so i'd think we should be testing on node 4 here also.